components/count-badge/count-badge-icon.js

```diff
@@ -0,0 +1,139 @@
+import '../colors/colors.js';
+import '../icons/icon.js';
+import { CountBadgeMixin, countBadgeStyles } from './count-badge-mixin.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { offscreenStyles } from '../offscreen/offscreen.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+import { styleMap } from 'lit-html/directives/style-map';
+
+class CountBadgeIcon extends CountBadgeMixin(RtlMixin(LitElement)) {
```

Comment:
I've tried splitting the badge into the two components and it seems to be working well. Still thinking about how to remove the forceTooltipOn property.

Reply:
Looks like the mixin already mixes in `RtlMixin` so I don't think you need it here (or in the non-icon one).

Reply:
Re-upping this ^ comment about `RtlMixin` not being necessary since `CountBadgeMixin` includes it.